src/Parser/B3dmParser.js

```diff
@@ -72,6 +72,31 @@ export function enableDracoLoader(path, config) {
     glTFLoader.setDRACOLoader(dracoLoader);
 }
 
+/**
+ * Enable KTX2 decoding for gltf.
+ * @param {string} path path to KTX2 library folder.
+ * @param {THREE.WebGLRenderer} renderer the threejs renderer
+ * This library is mandatory to load b3dm and gltf with KTX2 compression.
+ *
+ * The KTX2 library files are in folder itowns/examples/libs/basis/.
+ * You must indicate this path when you want to enable KTX2 decoding.
+ *
+ * For more information about KTX2, read /itowns/examples/libs/basis/README.md.
+ *
+ * @example <caption>Enable ktx2 decoder</caption>
+ * // if you copied /itowns/examples/libs/draco/ to the root folder of your project,you can set the path to './'.
+ * itowns.enableKtx2Loader('./', view.mainLoop.gfxEngine.renderer);
```

Comment:
Maybe we could ease access to the renderer to the user (scope for another PR). WDYT ? Other than that I understand from `KTX2Loader` the need of passing a `renderer`, but would it be possible to access it in the parser in some way ?

Reply:
Regarding the method to access the renderer, I thought the same and implemented it (among other similar methods) in #2174 :slightly_smiling_face: 

Regarding accessing the renderer in the parser, I think we should not access it in the parser and the parser should not know the renderer because it only needs to parse and not to render. However we could avoid passing it as an argument and either store it as a global variable and retrieve it in `enableKtx2Loader` or implement a service that stores it and provides a getter (e.g. `RendererService` and do a `RendererService.getRenderer()` to get it). This pattern is not used in itowns and I'm not sure we should introduce it just for this case but I like it more than storing a global variable. Anyway I think it should be done in another PR. 

Moreover, I think these two methods (`enableKtx2Loader` and `enableDracoLoader`) should be defined in `C3DTilesLayer` for more clarity (especially in the documentation). However they need to know the `gltfParser` and the `legacyGltfParser` which is currently defined in the `B3dmParser`. Same here, we could store them as global variables and use them in the `B3dmParser` or implement a service to store and retrieve them. This should also be done later I think, especially after #2124 that introduces a single class for gltf parsing.

Sorry for the long answer :sweat_smile:

Reply:
I can see how moving `enableKtx2Loader` and `enableDracoLoader` in `C3DTilesLayer` would ease their use, but I'm not sure I agree in terms of architecture. Those methods are "global" in a sense that they are not related to a specific 3d-tiles layer. Perhaps using them from a specific layer could be misleading.

Reply:
yes you're right